helm/tests/api/deployment_test.yaml

```diff
@@ -112,3 +112,19 @@ tests:
               items:
                 - key: licensekey
                   path: license.key
+
+  - it: Deploy with podSecurityContext
+    template: api/api-deployment.yaml
+    set:
+      api:
+        deployment:
+          podSecurityContext:
+            fsGroup: 1001
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Deployment
+      - equal:
+          path: spec.template.spec.securityContext.fsGroup
+          value: 1001
```

Comment:
**todo**: Can we have a test case where nothing is specified for `podSecurityContext` please?